tests/system/conftest.py

```diff
@@ -165,12 +167,20 @@ def signing_bucket(storage_client, signing_bucket_name):
     _helpers.delete_bucket(bucket)
 
 
-@pytest.fixture(scope="session")
+@pytest.fixture(scope="function")
 def default_ebh_bucket_name():
-    return _helpers.unique_name("gcp-systest-default-ebh")
+    # Keep track of how many ebh buckets have been created so we can get a
+    # clean one each rerun. "unique_name" is unique per test iteration, not
+    # per test rerun.
+    global ebh_bucket_iteration
+    ebh_bucket_iteration += 1
```

Comment:
Why not simply use `uuid.uuid4().hex[:10]` to have unique names without tracking?

Reply:
I figured if we need to examine the artifacts or their logs later, having them in order would make it easier to determine which one we're looking for.